src/SecurityChecks.js

```diff
@@ -0,0 +1,165 @@
+import { getTrigger } from './triggers.js';
+import url from 'url';
+import Parse from 'parse/node';
+export async function securityChecks(req) {
```

Comment:
Can you refactor the whole file and group the security checks into functions, according to what they check. For example all DB related checks grouped into a function. I imagin this file to grow more, so this would help with oversight. Something like:

function securityChecks() {

  await checkDatabase();
  await checkCLP();
  ...
}

Reply:
Ok, how does that look now?

Reply:
Please see the comments in the following review.